### PR TITLE
Feat: wishlist added

### PR DESCRIPTION
**Pull request by Rohanmond**
Screenshot 2022-02-11 at 2 13 48 AM
Added wishlist page
File Structure:
Pages=>
    => WishList
        =>  wishlist.html
          wishlist.css

### CODE REVIEW

File: pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.wishlist-container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+.wishlist-main-heading {
+  padding: 1.75rem;
+}
+.wishlist-main-cart-container {
+  margin-left: auto;
```
Comment (Parul1999) [inline pages/wishlist/wishlist.css]: Can't we write margin: 0 auto ?

File: pages/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FashiFy | Wishlist</title>
+    <link rel="stylesheet" href="../../style.css" />
+  </head>
+  <body>
+    <div class="wishlist-container">
+      <nav class="navigation">
+        <div class="nav-left">
```
Comment (Parul1999) [inline pages/wishlist/wishlist.html:13]: Having two divs , why is it necessary ? One div can be used.

**Review by anuj-kumary — APPROVED** [review]

File: pages/wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FashiFy | Wishlist</title>
+    <link rel="stylesheet" href="../../style.css" />
+  </head>
+  <body>
+    <div class="wishlist-container">
```
Comment (Parul1999) [inline pages/wishlist/wishlist.html:11]: Can't nav act as the container itself?

File: pages/wishlist/wishlist.css (hunk 1 above)
Comment (Rohanmond) [reply pages/wishlist/wishlist.css]: Thanks..It is fixed now

File: pages/wishlist/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+      <nav class="navigation">
+        <div class="nav-left">
+          <div class="nav-logo-container">
+            <a href="#">
+              <img
+                class="hero-logo"
+                src="../../src/logo/Fashify Transparent pr.png"
+                alt="nav image"
```
Comment (Parul1999) [inline pages/wishlist/wishlist.html]: In alt have a name which user understands, as screen-reader will read nav image which a user can't understand

File: pages/wishlist/wishlist.html (hunk 3 above)
Comment (Rohanmond) [reply pages/wishlist/wishlist.html:11]: Actually I put outer div height as 100vh and put display as flex and nav and main are the child of the outer div

File: pages/wishlist/wishlist.html (hunk 4 above)
Comment (Rohanmond) [reply pages/wishlist/wishlist.html]: Thanks.It is fixed now

File: pages/wishlist/wishlist.html (hunk 2 above)
Comment (Rohanmond) [reply pages/wishlist/wishlist.html:13]: It is possible in wishlist but to maintain the consistency with other pages I have to put nav inside the outer container where for outer container I put height as 100vh

**Review by anuj-kumary — COMMENTED** [review]
A minimal changes just increase a width of your card then it look nyc

**Comment by Rohanmond** [comment]
> A minimal changes just increase a width of your card then it look nyc

it is fixed

**Pull request merged**